test/integration/kubernetes_deploy_test.rb

```diff
@@ -550,6 +550,32 @@ def test_success_detection_tolerates_out_of_band_deployment_scaling
     assert_equal true, success, "Failed to deploy deployment with dynamic replica count"
   end
 
+  def test_successful_daemon_set_deploy
+    assert deploy_fixtures("hello-cloud", subset: ["daemon_set.yml"])
+
+    assert_logs_match_all([
+      'Successfully deployed 1 resource',
+      '1 currentNumberSchedule, 1 desiredNumberSchedule, 1 numberRead'
```

Comment:
Since these tests are expensive, I'd recommend adding this assertion to the group at L17 (including a resource type prefix like the others there have) instead of having a separate test.